src/rhsmlib/facts/cloud_facts.py

```diff
@@ -73,9 +85,25 @@ def get_aws_instance_id(self):
             else:
                 return {}
 
-    def parse_content(self, content):
+    @staticmethod
+    def get_aws_metadata():
+        """
+        Try to get AWS metadata
+        :return: http response
+        """
+        conn = httplib.HTTPConnection(AWS_INSTANCE_IP, timeout=AWS_INSTANCE_TIMEOUT)
```

Comment:
Shouldn't we use the values from aws.conf here (or I'm guessing in the future collector).

Reply:
I have plan to use code from `CloudCollector`. So, this code will change.

Reply:
Sounds good